src/charts/scatter-plot.js

```diff
@@ -902,8 +896,9 @@ define(function(require) {
          * @param {Number}
          * @private
          */
-        function setCrossHairLinesStatus(status = 0) {
+        function setCrossHairComponentStatus(status = 0) {
```

Comment:
A bit offtopic, but I would prefer a more explicit API for this, maybe showCorssHairComponent/hideCrossHairComponent?

Reply:
That makes sense! I will make more explicit functions in next PRs. Thanks for the suggestion!